src/main/java/step4/model/LottoGenerator.java

```diff
@@ -0,0 +1,26 @@
+package step4.model;
+
+import step4.constant.LottoConstant;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGenerator {
+
+    private static final List<LottoNumber> lottoCandidateNumbers;
+
+    static {
+        lottoCandidateNumbers = IntStream.range(LottoConstant.LOTTO_MIN_NUM, LottoConstant.LOTTO_MAX_NUM + 1)
+                .mapToObj(LottoNumber::new).collect(Collectors.toList());
```

Comment:
다음 메서드를 활용해보시면 1을 추가하실 필요는 없을거에요!! 
참고만해주세요 🙏

```suggestion
        lottoCandidateNumbers = IntStream.rangeClosed(LottoConstant.LOTTO_MIN_NUM, LottoConstant.LOTTO_MAX_NUM)
                .mapToObj(LottoNumber::new).collect(Collectors.toList());
```

Reply:
앗 이런 함수가 있었군요 감사합니다!